armi/reactor/tests/test_reactors.py

```diff
@@ -179,7 +179,6 @@ def loadTestReactor(
         assemblies.setAssemNumCounter(assemNum)
         settings.setMasterCs(o.cs)
         o.reattach(r, o.cs)
-        r.sort()
```

Comment:
these are not needed since its sorted on initialization as the test below shows.